### PR TITLE
Click.option-s should have help text

### PR DESCRIPTION
**Pull request by ulziibay**
## What?
Add `help` text to click.option

## Why?
Click.option should ideally have a `help` text defined to be useful.

### CODE REVIEW

File: wandb/cli.py [hunk 1]
```diff
@@ -775,7 +775,7 @@ def run(ctx, program, args, id, resume, dir, configs, message, name, notes, show
 @cli.command(context_settings=RUN_CONTEXT, name="docker-run")
 @click.pass_context
 @click.argument('docker_run_args', nargs=-1)
-@click.option('--help')
+@click.option('--help', is_flag=True, defualt=False, help='Show docker run helper.')
```
Comment (raubitsj) [inline wandb/cli.py]: There appears to be a typo with "defualt" here.

Thanks for finding this.
I think a better fix is to add help="Wrap docker run with W&B api key" to the `@cli.command()` line and remove the `help` flag altogether.

Comment (ulziibay) [reply wandb/cli.py]: `help` flag is still used in the code for printing docker help text, so lets keep it for now. I addressed the other two comments.

Comment (vanpelt) [reply wandb/cli.py]: `defualt` => `default`

Comment (ulziibay) [reply wandb/cli.py]: yes its addressed in `25e2466`

**Pull request closed without merging**